.github/workflows/homebrew.yml

```diff
@@ -0,0 +1,17 @@
+name: homebrew
+
+on:
+  push:
+    tags: 'v*'
+
+jobs:
+  homebrew:
+    name: Bump Homebrew formula
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v1
+        with:
+          # A PR will be sent to github.com/Homebrew/homebrew-core to update this formula:
+          formula-name: tflint
+        env:
+          COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
```

Comment:
Does this secret already exist?

Reply:
Not quite yet, I am working with @wata727 to set this up.

Reply:
Yeah, just I completed this setup.

Reply:
Thanks!!